README.md

```diff
@@ -0,0 +1,48 @@
+1. Download the `master` branch of _this template_ as a .zip and unzip it.
+1. Because you cannot make a new branch until you have an initial commit to your own new repo's `master` branch, copy this template's `license.txt` file and commit it to your repo's `master` branch.
+1. Create a new branch called `initial` or `v1` or whatever is appropriate and check out this branch.
+1. Place the remainder of this template's files in your repo's new branch.
+1. Find and replace these strings throughout all file names and file contents:
+    1. Extension plugin's slug: `tribe-ext-extension-template`
+    1. Extension plugin's slug (yes, again): `match-the-plugin-directory-name`
+    1. `[Base Plugin Name] Extension: [Extension Name]`:
+        1. `[Base Plugin Name]` should be something like `The Events Calendar`
+        1. `[Extension Name]` should be the name of your extension, such as `Remove Export Links`
+    1. `[Extension Description]` with your extension plugin's short description.
+    1. PHP namespace: `Tribe\Extensions\Example`
+    1. Extension class (must be unique among all extensions): `Tribe__Extension__Example`
+    1. 
+1. Set `Plugin URI` accordingly.
+1. Set `GitHub Plugin URI` accordingly.
+1. Set `Author:` and `Author URI:` accordingly if not being authored by Modern Tribe.
+1. Set the `== Description ==` section of `readme.txt`
+1. Add your required plugin(s) via `$this->add_required_plugin()`.
+    1. You should have at least one required plugin and/or conditionally-required plugin(s), making use of `Tribe__Dependency`.
+    1. If you're requiring a specific version, add a small code comment why this version is required. You should probably also mention this in `readme.txt`.
+1. Remove the starter bits that don't apply to your extension, such as required PHP version comments.
+    1. Look for `TODO` comments to identify some things to modify or remove, as applicable. For example, remove the entire `src` directory and the `Tribe__Autoloader` code if you do not have any settings to add to the admin UI and do not need the autoloader for any other purpose.
+    1. It might be advisable to leave the PHP version checking logic (and matching part of readme.txt) unmodified until one of your last commits to ensure you correctly determine the required version.
+1. Add your filters and actions in the `init()` method, creating your custom methods as needed.
+1. Remove all unused/unnecessary code and comments, as this template may handle edge cases that do not apply to your extension.
+1. Sanitize all input and escape all output, as appropriate.
+1. Double-check your plugin's slug, name, and descriptions still apply to the end result of what your code actually accomplishes.
+1. Set the `== Changelog ==` section of `readme.txt`.
+1. Delete this `README.md` file.
+1. Zip and release your plugin!
```

Comment:
I love that we started with a step to _Create a new branch called `initial` or `v1` or whatever is appropriate_ as it helps us set things up in a way that will make pull requests/initial code review easier, but it seems like we get to _Zip and release your plugin!_ without an actual code review step.

Especially for new extension authors, that's important — can we add it?